test/spacy/smoke_test.clj

```diff
@@ -0,0 +1,43 @@
+(ns spacy.smoke-test
+  (:require [clojure.test :refer :all]
+            [clojure.java.io :as io]
+            [com.stuartsierra.component :as component]
+            [aleph.http :as http]
+            [spacy.system :as sys]))
+
+(def port (+ 39215 (rand-int 1000)))
+
+(defn url [path]
+  (format "http://localhost:%s%s" port path))
+
+(defn get-body [path]
+  (-> path url http/get deref :body slurp))
+
+(def dbname
+  (format "%s/spacy-test-%s.db"
+          (System/getProperty "java.io.tmpdir")
+          (rand-int 1000)))
+
+(deftest integration-test
+  (let [system (-> {:webserver {:port port}
+                    :crux {:db-spec {:dbtype "sqlite"
+                                     :dbname dbname}}}
+                   (sys/system)
+                   (component/start))]
+    (try
+
+      (Thread/sleep 100) ;; wait for DB seeding
+
+      @(http/post (url "/dezember-2020-strategie-event/submit-session")
```

Comment:
Where does the `@` come from? Is this Clojure syntax?

Reply:
Yes, that's a shorthand for `deref`. `@x` is equivalent to `(deref x)`.

Reply:
🤦‍♀️ I should have known that.

I guess I only ever use `deref` with atoms, and it didn't look like an atom to me.